src/rpc/rawtransaction.cpp

```diff
@@ -218,6 +218,252 @@ UniValue getrawtransaction(const UniValue &params, bool fHelp)
+            "       {\n"
+            "         \"value\" : x.xxx,            (numeric) The value in " +
+            CURRENCY_UNIT +
+            "\n"
+            "         \"n\" : n,                    (numeric) index\n"
+            "         \"scriptPubKey\" : {          (json object)\n"
+            "           \"asm\" : \"asm\",          (string) the asm\n"
+            "           \"hex\" : \"hex\",          (string) the hex\n"
+            "           \"reqSigs\" : n,            (numeric) The required sigs\n"
+            "           \"type\" : \"pubkeyhash\",  (string) The type, eg 'pubkeyhash'\n"
+            "           \"addresses\" : [           (json array of string)\n"
+            "             \"bitcoinaddress\"        (string) bitcoin address\n"
+            "             ,...\n"
+            "           ]\n"
+            "         }\n"
+            "       }\n"
+            "      ,...\n"
+            "      ],\n"
+            "    \"blockhash\" : \"hash\",   (string) the block hash\n"
+            "    \"confirmations\" : n,      (numeric) The confirmations\n"
+            "    \"time\" : ttt,             (numeric) The transaction time in seconds since epoch (Jan 1 1970 GMT)\n"
+            "    \"blocktime\" : ttt         (numeric) The block time in seconds since epoch (Jan 1 1970 GMT)\n"
+            "  },\n"
+            "  ...\n"
+            "}\n"
+            "\nExamples:\n" +
+            HelpExampleCli("getrawtransactions", "\"hashblock\"") +
+            HelpExampleCli("getrawtransactions", "\"hashblock\" 1") +
+            HelpExampleRpc("getrawtransactions", "\"hashblock\", 1"));
+
+    uint256 hashBlock = ParseHashV(params[0], "parameter 1");
+
+    bool fVerbose = false;
+    if (params.size() > 1)
+        fVerbose = (params[1].get_int() != 0);
+
+    CBlockIndex *pblockindex = nullptr;
+    pblockindex = LookupBlockIndex(hashBlock);
+    if (!pblockindex)
+        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Block not found");
+
+    CBlock block;
+    if (!ReadBlockFromDisk(block, pblockindex, Params().GetConsensus()))
+        throw JSONRPCError(RPC_INTERNAL_ERROR, "Can't read block from disk");
+
+    UniValue resultSet(UniValue::VOBJ);
+    for (auto tx : block.vtx)
+    {
+        string strHex = EncodeHexTx(*tx);
+
+        if (!fVerbose)
+        {
+            resultSet.pushKV(tx->GetHash().GetHex(), strHex);
+            continue;
+        }
+
+        UniValue result(UniValue::VOBJ);
+        result.pushKV("hex", strHex);
+        TxToJSON(*tx, block.GetHash(), result);
+        resultSet.push_back(result);
+    }
+    return resultSet;
+}
+
+UniValue getrawtransactionssince(const UniValue &params, bool fHelp)
+{
+    if (fHelp || params.size() < 1 || params.size() > 3)
+        throw runtime_error(
+            "getrawtransactionssince \"blockhash\" ( verbose ) ( count )\n"
+            "\nReturn the raw transaction data for a <count> blocks starting with blockhash and moving towards the "
```

Comment:
remove extra "a"

Is "count" blocks what the user wants, or does he really want approximately N bytes of data?  If this API is meant to chunk the data, it would make more sense to specify N bytes and we give him M complete blocks such that Len(message(M)) >= N.

Reply:
This was implemented as requested. This was a way to batch multiple getrawblocktransactions into one call. 
Overall the process moved from getrawtransaction for every tx in a block every block. to getrawblocktransactions for one block or getrawtransactionssince for multiple blocks. 

although i see now how the naming convention is probably not the best to describe the use case